src/sentry/api/endpoints/project_rules.py

```diff
@@ -113,6 +113,7 @@ def post(self, request: Request, project) -> Response:
             RuleActivity.objects.create(
```

Comment:
does workflow own this endpoint?

Reply:
Looks like no team owns this file in `CODEOWNERS` but workflow should own all the alert creation endpoints